dbms/src/Columns/ColumnVector.cpp

```diff
@@ -339,7 +339,7 @@ ColumnPtr ColumnVector<T>::index(const IColumn & indexes, size_t limit) const
 template <typename T>
 ColumnPtr ColumnVector<T>::replicate(const IColumn::Offsets & offsets) const
 {
-    size_t size = data.size();
+    const size_t size = data.size();
```

Comment:
I doubt this is significant.

Reply:
So the hypothesis is that the compiler is thinking that the `size` variable can alias with `curr` if it's a pointer to the same type? Maybe adding __restrict to curr pointer will also help.

Reply:
This change was meant to be solely cosmetic! And I apologize for the obvious confusion that it has introduced